### ISSUE TITLE
error in output - `-

### ISSUE THREAD

**Issue by daniejstriata** [jtesta/ssh-audit#108]
When ssh-audit completed I see under the ciphers section text that does not match formatting. Not sure what is wrong here:
`- [info] default cipher since OpenSSH 6.9.

![image](https://user-images.githubusercontent.com/13314239/118480754-f1dcb600-b712-11eb-8751-e30b89a45d60.png)

ssh-audit v2.4.0 installed from the snap package.

**Comment by jtesta** [comment]
The 'default cipher' line is formatted that way to tell the user that it belongs to the entry above it (in this case, to the chacha20 algorithm).  The back-tick character is used since its the closest ascii character to a partly slanted line.

Here's a crudely-drawn image of what the two comments are trying to do:

![image](https://user-images.githubusercontent.com/2982011/118517340-1d58a400-b705-11eb-9ce2-7eb56242b5f7.png)

**Comment by daniejstriata** [comment]
Can you make it green like the others as it is not an error? Further, as it is the only default, why not print it first, as the default and then the rest that is available? At least in the same colour? Looking at it, I think it was the white coloured text then the grave accent that made me think there is something missing.

Some options come to mind for your consideration.

```
# encryption algorithms (ciphers)
## Default
(enc) chacha20-poly1305@openssh.com         -- [info] default cipher since OpenSSH 6.9.

## Available
(enc) chacha20-poly1305@openssh.com         -- [info] available since OpenSSH 6.5
(enc) aes256-gcm@openssh.com                -- [info] available since OpenSSH 6.2
(enc) aes128-gcm@openssh.com                -- [info] available since OpenSSH 6.2
(enc) aes256-ctr                            -- [info] available since OpenSSH 3.7, Dropbear SSH 0.52
(enc) aes192-ctr                            -- [info] available since OpenSSH 3.7
(enc) aes128-ctr                            -- [info] available since OpenSSH 3.7, Dropbear SSH 0.52
```
or

```
# encryption algorithms (ciphers)
(enc) chacha20-poly1305@openssh.com         -- [info] available since OpenSSH 6.5 – Default cipher since OpenSSH 6.9.
(enc) aes256-gcm@openssh.com                -- [info] available since OpenSSH 6.2
(enc) aes128-gcm@openssh.com                -- [info] available since OpenSSH 6.2
(enc) aes256-ctr                            -- [info] available since OpenSSH 3.7, Dropbear SSH 0.52
(enc) aes192-ctr                            -- [info] available since OpenSSH 3.7
(enc) aes128-ctr                            -- [info] available since OpenSSH 3.7, Dropbear SSH 0.52
```
